### PR TITLE
PAAS-1479: Add/fix datadog service tag for ipsec, proxysql and haproxy

### PR DESCRIPTION
**Pull request by amitc-jahia**
JIRA issue: https://jira.jahia.org/browse/PAAS-1479

Short description:

### CODE REVIEW

File: migrations/v4/migration-to-v4.yml [hunk 1]
```diff
@@ -143,3 +144,38 @@ actions:
...
+          exit 0
+        fi
+        sed -i "/source: ipsec/a\\$insert_line" /etc/datadog-agent/conf.d/strongswan_connections_status.yaml
+      user: root
+
+  addDatadogServiceTagProxySql:
+    - cmd[${this}]: |-
+        insert_line='    service: ${env.shortdomain}'
+        if (grep -qPzo "$insert_line" /etc/datadog-agent/conf.d/proxysql.d/conf.yaml);
+        then
```
Comment (Ronan-Gorain) [inline migrations/v4/migration-to-v4.yml:167]: same line as the if to get the same syntax as other addDatadogServiceTagXXXX :police_car:  :)

File: migrations/v4/migration-to-v4.yml [hunk 2]
```diff
@@ -143,3 +144,38 @@ actions:
...
+      user: root
+
+  fixDatadogServiceTagHAProxy:
+    - cmd[${this}]: |-
+        insert_line='service: ${env.shortdomain}
+        if (grep -qPzo "$insert_line" /etc/datadog-agent/conf.d/haproxy.d/conf.yaml); then
+          echo "Correct Service tag already there"
+          exit 0
+        fi
+        sed -i "s/service: haproxy/\\$insert_line/' /etc/datadog-agent/conf.d/haproxy.d/conf.yaml
```
Comment (Ronan-Gorain) [inline migrations/v4/migration-to-v4.yml:181]: user: root missing

File: migrations/v4/migration-to-v4.yml (hunk 1 above)
Comment (Ronan-Gorain) [inline migrations/v4/migration-to-v4.yml:167]: As there is another CR, take this opportunity to move the "then" at the end of the if line :)

**Review by Ronan-Gorain — CHANGES_REQUESTED** [review]
user: root missing in   fixDatadogServiceTagHAProxy

**Comment by amitc-jahia** [comment]
Changes made as requested. Please review.

**Pull request merged**